rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/activityFilter.vue

```diff
@@ -91,6 +92,7 @@
                   {{ $t("jobquery.title.jobIdFilter") }}
                 </label>
                 <input
+                  ref="jobIdFilter"
```

Comment:
please remove the refs that aren't being used, it's preferable to use data-testids for tests whenever it's possible